challengeMk2.Android/Properties/AndroidManifest.xml

```diff
@@ -1,6 +1,6 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <manifest xmlns:android="http://schemas.android.com/apk/res/android" android:versionCode="1" android:versionName="1.0" package="mi8.challengemk2">
-	<uses-sdk android:minSdkVersion="21" android:targetSdkVersion="28" />
+	<uses-sdk android:minSdkVersion="21" android:targetSdkVersion="23" />
```

Comment:
A `targetSdkVersion` of 23 seems very low (it means you app is targeting Android 6.0) ... I would say `minSdkVersion` should be 23, and `targetSdkVersion` should be whatever is the latest SDK is installed on your machine.

Reply:
It was to test the app on the old phone (the Motorola one, iirw) at the office. I have forgot to reverse it after testing the app.
I reverse it to v28.